api-test/ApiConfig.postman_collection.json

```diff
@@ -3779,6 +3779,438 @@
 					}
 				}
 			]
+		},
+		{
+			"name": "api test suite - configuration",
+			"item": [
+				{
+					"name": "getConfigurationKeys",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"getConfigurationKeys\", function () {",
+									"  pm.response.to.have.status(200);",
+									"  const responseJson = pm.response.json();",
+									"  pm.expect(responseJson.configuration_keys).to.be.exist",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{api-config-host}}/{{api-config-base-path}}/configuration/keys",
+							"host": [
+								"{{api-config-host}}"
+							],
+							"path": [
+								"{{api-config-base-path}}",
+								"configuration",
+								"keys"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "createConfigurationKey",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"createConfigurationKey\", function () {",
+									"  pm.response.to.have.status(201);  ",
+									"  ",
+									"  const requestJson = JSON.parse(pm.request.body.raw);",
+									"  const responseJson = pm.response.json();",
+									"  pm.expect(responseJson.config_category).to.eql(requestJson.config_category);",
+									"  pm.expect(responseJson.config_key).to.eql(requestJson.config_key);",
+									"  pm.expect(responseJson.config_value).to.eql(requestJson.config_value);",
+									"  pm.expect(responseJson.config_description).to.be.exist;",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
```

Comment:
for k6 conversion it is better to make explicit the `Content-Type` header in POST and PUT operations.